### PR TITLE
RHIDP-3334: apiversion and kind update

### PR DESCRIPTION
**Pull request by pabel-rh**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.2.x, 1.3.x, main

**Issue:** RHIDP-3334

**Link to docs preview:**

**Reviews:**
- [x] SME: @rm3l 
- [x] QE: @josephca 
- [x] Docs review: @jmagak 
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

File: assemblies/assembly-add-custom-app-file-openshift.adoc [hunk 1]
```diff
@@ -5,8 +5,8 @@ To access the {product}, you must add a custom application configuration file to
 
 [source,yaml,subs="attributes+"]
 ----
-kind: ConfigMap
-apiVersion: v1
+apiVersion: rhdh.redhat.com/v1alpha1
+kind: Backstage
```
Comment (rm3l) [inline assemblies/assembly-add-custom-app-file-openshift.adoc:9]: I don't think this is the YAML that needs to be changed. This is a valid ConfigMap manifest and seems to be common to the procedures for both Helm and Operator. The error was specific to the Operator.

image

**Review by rm3l — CHANGES_REQUESTED** [review]

**Review by rm3l — APPROVED** [review]
Thanks @pabel-rh !

**Review by jmagak — COMMENTED** [review]
LGTM

**Pull request merged**